db/change_cache_test.go

```diff
@@ -1185,3 +1187,114 @@ func readNextFromFeed(feed <-chan (*ChangeEntry), timeout time.Duration) (*Chang
 	}
 
 }
+
+
+// This was added to increase code coverage as part of SG #2633
+// It was artifically derived to pick one arbitrarily chosen code block in the change set mentioned
+// in that issue, to try to exercise that code.  If that code block is commented, it does catch
+// error which demonstrates that the code block is useful.
+// This test could also serve as example code for other tests that test the changeCache (channel cache) directly.
+//
+// 1. Create a doc + feed event with unusued sequences 10 and 11
+// 2. Create another doc + feed event that tries to reuse sequence 10 with a doc in channel ABC -- it should be ignored by the change cache
+// 3. Detect whether the 2nd was ignored using an onChange listener callback and make sure it was not added to the ABC channel
```

Comment:
This might provide code coverage, but I don't see how it actually covers the fix associated with #2633.  This approach (manually sending feed events) is a good way to test this, though.

I would have expected that test to look something like:
1. Create doc1 w/ unused sequences 1, actual sequence 3.
2. Create doc2 w/ sequence 2, channel ABC
3. Send feed event for doc2.  This won't trigger onChange, as buffering is waiting for seq 1
4. Send feed event for doc1.  This should trigger caching for doc2, and trigger onChange for channel ABC.  

Previous bug was that onChange for channel ABC wasn't sent.